catalogue/webapp/components/PhysicalItemDetails/PhysicalItemDetails.tsx

```diff
@@ -5,15 +5,26 @@ import Space from '@weco/common/views/components/styled/Space';
 import { classNames, font } from '@weco/common/utils/classnames';
 import IsArchiveContext from '@weco/common/views/components/IsArchiveContext/IsArchiveContext';
 
-const Wrapper = styled.div`
-  border-bottom: 1px solid ${props => props.theme.color('pumice')};
-  margin-bottom: 20px;
-`;
-
 const Row = styled(Space).attrs({
   v: { size: 'm', properties: ['margin-bottom'] },
 })``;
 
+const Wrapper = styled(Space).attrs({
+  v: { size: 'm', properties: ['margin-bottom', 'padding-bottom'] },
+})<{ noUnderline: boolean }>`
+  border-bottom: 1px solid ${props => props.theme.color('pumice')};
```

Comment:
Would it be easier to read as `underline` and then have `underline={!isLast}` avoiding the need to override?